src/Synapse/Synapse/Models/ManagementModels/Workspace/PSDataLakeStorageAccountDetails.cs

```diff
@@ -18,15 +18,32 @@ namespace Microsoft.Azure.Commands.Synapse.Models
 {
     public class PSDataLakeStorageAccountDetails
     {
-        public string DefaultDataLakeStorageAccountUrl { get; set; }
-
-        public string DefaultDataLakeStorageFilesystem { get; set; }
-
-
```

Comment:
It will be a breaking change, we often do not change the property name if it's not really needed